instrumentation/slowrendering/src/test/java/io/opentelemetry/android/instrumentation/slowrendering/SlowRenderListenerTest.kt

```diff
@@ -0,0 +1,289 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.instrumentation.slowrendering
+
+import android.app.Activity
+import android.content.ComponentName
+import android.os.Build
+import android.os.Handler
+import android.view.FrameMetrics
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.opentelemetry.api.common.AttributeKey
+import io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions
+import io.opentelemetry.sdk.testing.junit4.OpenTelemetryRule
+import io.opentelemetry.sdk.trace.data.SpanData
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.ThrowingConsumer
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Answers
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers
+import org.mockito.Captor
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnit
+import org.mockito.junit.MockitoRule
+import org.mockito.stubbing.Answer
+import org.robolectric.annotation.Config
+import java.time.Duration
+import java.util.concurrent.Executors
+import java.util.concurrent.ScheduledExecutorService
+import java.util.concurrent.TimeUnit
+import java.util.stream.Collectors
+import java.util.stream.Stream
+
+@RunWith(AndroidJUnit4::class)
+@Config(sdk = [Build.VERSION_CODES.N])
+class SlowRenderListenerTest {
+    @get:Rule
+    var otelTesting: OpenTelemetryRule = OpenTelemetryRule.create()
+
+    @get:Rule
+    var mocks: MockitoRule = MockitoJUnit.rule()
+
+    @Mock
+    lateinit var frameMetricsHandler: Handler
+
+    @Mock(answer = Answers.RETURNS_DEEP_STUBS)
+    lateinit var activity: Activity
+
+    @Mock
+    lateinit var frameMetrics: FrameMetrics
+
+    @Mock
+    internal lateinit var jankReporter: JankReporter
+    private lateinit var executorService: ScheduledExecutorService
+
+    @Captor
+    internal lateinit var activityListenerCaptor: ArgumentCaptor<PerActivityListener>
+
+    @Before
+    fun setup() {
+        executorService = Executors.newSingleThreadScheduledExecutor()
+        val componentName = ComponentName("io.otel", "Komponent")
+        Mockito.`when`(activity.componentName).thenReturn(componentName)
+    }
+
+    @Test
+    fun add() {
+        val testInstance =
+            SlowRenderListener(
+                jankReporter,
+                executorService,
+                frameMetricsHandler,
+                Duration.ZERO,
+            )
+
+        testInstance.onActivityResumed(activity)
+
+        Mockito
+            .verify(activity.window)
+            .addOnFrameMetricsAvailableListener(
+                activityListenerCaptor.capture(),
+                ArgumentMatchers.eq(frameMetricsHandler),
+            )
+        Assert.assertEquals(
```

Comment:
We prefer the assertj style assertions, can you modify this to use assertj please?